dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -97,14 +106,30 @@ def debug_str(self):
 
         return s
 
+    def check_import(self, lib_path):
+        import tensorflow as tf
+        assert can_import_dali()  # DALI symbols need to be loaded
+        print("Importing the TF library to check for errors")
+        try:
+            tf.load_op_library(lib_path)
+            return True
+        except Exception as e:
+            print("Failed to import TF library: ", str(e))
+            return False
+
     def install_prebuilt(self):
         assert(self.can_install_prebuilt)
         assert(self.prebuilt_plugin_best_match is not None)
         assert(self.plugin_name is not None)
         print("Tensorflow was built with g++ {}, providing prebuilt plugin".format(self.tf_compiler))
-        plugin_dest =  os.path.join(self.plugin_dest_dir, self.plugin_name)
-        print("Copy {} to {}".format(self.prebuilt_plugin_best_match, self.plugin_dest_dir))
-        copyfile(self.prebuilt_plugin_best_match, plugin_dest)
+        if self.check_import(self.prebuilt_plugin_best_match):
```

Comment:
Even if `best_version == self.tf_version` you still wants to check import, righ?

Reply:
In such case installing DALI together with the plugin will just fail, as there is no DALI available and you cannot `check_import` that seems to be executed in every code path now.

Reply:
That's right, check_import is executed even for exact TF version. Then we should only check import if we can. I'll correct